app/controllers/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find params[:id]
+  end
+
   def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "welcome"
```

Comment:
Em, can you use I18n here
[http://guides.rubyonrails.org/i18n.html](http://guides.rubyonrails.org/i18n.html)